components/Authentication/AuthenticationForm.tsx

```diff
@@ -0,0 +1,108 @@
+import Link from 'next/link'
+
+import { Input } from 'components/design/Input'
+import { FormState } from 'components/Authentication/FormState'
+import { ValidatorSchema } from 'hooks/useFastestValidator'
+import { AuthenticationProps } from '.'
+import { AuthenticationFormLayout } from './AuthenticationFormLayout'
+import { useForm } from 'hooks/useForm'
+
+export const emailSchema: ValidatorSchema = {
+  email: {
+    type: 'email',
+    empty: false,
+    trim: true,
+    messages: {
+      emailEmpty: 'Oops, this field is required 🙈.',
```

Comment:
Why not `empty` and `invalid`? We already know we are in the `email` field
```suggestion
      empty: 'Oops, this field is required 🙈.',
      invalid: 'Mmm… It seems that this email is not valid 🤔.',
```

Reply:
You're right, I agree with you, but we can't choose that, since it's how we describe the validation schema for `fastest-validator` library.